packages/driver/src/cy/snapshots.ts

```diff
@@ -49,11 +51,21 @@ export const create = ($$: $Cy['$$'], state: StateFunc) => {
 
     return $$('iframe').each((idx, iframe) => {
       const $iframe = $(iframe)
+      const props = {
+        id: iframe.id,
+        class: iframe.className,
+        style: iframe.style.cssText,
+      }
+      const key = JSON.stringify({ ...props })
+
+      iframePlaceholders.has(key)
```

Comment:
What is this supposed to do? It is a no-op - should this be a different method? Or should it be `iframePlaceholders.add(key)`

You are using a Map so should it be `iframePlaceholders.set(key, props)`? Also won't you need to save the snapshot, too (`iframe`) in the `props`?

Reply:
I think I've messed up something right before creating a commit. This line 61 shouldn't exist.

I am saving iframe placeholder for the snapshot in line 105. So that next time (line 81) I can ask the map if there I've already created placeholder for identical iframe, if true, I try to re-use it.

I don't know yet what should be included in key-creation. At that moment I was just prototyping and was trying to see if I'm able to 'skip' creating at least some iframe plceholders. In next iteration I would probably try to be more precise and construct key in a way that makes sure only identical iframes are replaced with placeholders from the map